doc/src/fix_shake.rst

```diff
@@ -152,17 +156,26 @@ for.
 
 ----------
 
-The *mol* keyword should be used when other commands, such as :doc:`fix deposit <fix_deposit>` or :doc:`fix pour <fix_pour>`, add molecules
+The *mol* keyword should be used when other commands, such as :doc:`fix
+deposit <fix_deposit>` or :doc:`fix pour <fix_pour>`, add molecules
 on-the-fly during a simulation, and you wish to constrain the new
 molecules via SHAKE.  You specify a *template-ID* previously defined
 using the :doc:`molecule <molecule>` command, which reads a file that
 defines the molecule.  You must use the same *template-ID* that the
 command adding molecules uses.  The coordinates, atom types, special
-bond restrictions, and SHAKE info can be specified in the molecule
-file.  See the :doc:`molecule <molecule>` command for details.  The only
+bond restrictions, and SHAKE info can be specified in the molecule file.
+See the :doc:`molecule <molecule>` command for details.  The only
 settings required to be in this file (by this command) are the SHAKE
 info of atoms in the molecule.
 
+The *kbond* keyword allows to set the restraint force constant when fix
+shake or fix rattle are used during minimization. In that case the
+constraint algorithms are *not* applied and restraint forces are used
+instead to help maintaining the geometries.  How well the geometries
+are maintained and how quickly a minimization will converge depends on
+the magnitude of the force constant (kbond).  If it is chosen too large
+the minimization may converge slowly.  The default is 1.0e6*k_B.
```

Comment:
now a different default?

Reply:
yes. k_B seems to be scaling somewhat similar than the magnitude of force constants for different units settings.
For `real` units that leads to kbond~=2000 which is ok for the systems I tested so far. Larger values like 5000 or 10000 maintain the geometries tighter, but lead to many more iterations and thus slow down the minimization a lot.
Also with minimizers like "fire" one needs to reduce the timestep with such large restraint forces or you risk lost bond atoms. So a not so tight default is a safer choice.